marketplace-service/src/main/java/com/axonivy/market/entity/productjsonfilecontent/Dependency.java

```diff
@@ -0,0 +1,13 @@
+package com.axonivy.market.entity.productjsonfilecontent;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Dependency {
```

Comment:
To make project be familiar with the old version, should we change its name to "maven artifact" since it can download from the maven repo & be used to install in the AI Designer.
It also be used in meta.json so consider changing the name of the package

Reply:
this model is created to map the properties with fields in the product.json. I think that it shouldn't be changed. It will be more easily for maintaining when checking the field after mapping